src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int BALL_COUNT = 100;
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(BALL_COUNT));
+        return ball;
+    }
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, you're creating a new instance of `ColorSupplier` and `Random` every time `getRandomBall()` is called. It would be better to create a single instance of each class and reuse them.